platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/wiring/PlatformSchedulersConfig.java

```diff
@@ -86,6 +86,7 @@
  *                                                          buffer
  * @param issDetectorSchedulerType                          the ISS detector scheduler type
  * @param issDetectorUnhandledCapacity                      number of unhandled tasks allowed for the ISS detector
+ * @param completeStateNotifierUnhandledCapacity            number of unhandled tasks allowed for the state completion notifier
```

Comment:
Now that this value is configured here, the class `WiringConfig` has no usages, and can be deleted.

Reply:
I noticed it still getting referenced but looking at it again, looks like it's good to go. Deleted.

Reply:
done